applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -26,10 +28,97 @@ LOG_MODULE_REGISTER(hw_codec, CONFIG_MODULE_HW_CODEC_LOG_LEVEL);
 #define VOLUME_ADJUST_STEP_DB 3
 #define BASE_10		      10
 
+ZBUS_SUBSCRIBER_DEFINE(volume_evt_sub, CONFIG_VOLUME_MSG_SUB_QUEUE_SIZE);
+
 static uint32_t prev_volume_reg_val = OUT_VOLUME_DEFAULT;
 
 static cs47l63_t cs47l63_driver;
 
+static k_tid_t volume_msg_sub_thread_id;
+static struct k_thread volume_msg_sub_thread_data;
+
+K_THREAD_STACK_DEFINE(volume_msg_sub_thread_stack, CONFIG_VOLUME_MSG_SUB_STACK_SIZE);
+
+/**
+ * @brief  Convert zbus volume to actual volume setting for HW codec.
+ *
+ * @note   The range for zbus volume is from 0 to 255 and the
+ *         range for HW codec volume is from 0 to 128, this function
+ *         converting the zbus volume to HW codec volume setting.
+ */
+static uint16_t zbus_vol_conversion(uint8_t volume)
+{
+	return (((uint16_t)volume + 1) / 2);
+}
+
+/* Handle volume events from zbus */
```

Comment:
```suggestion
/* Handle volume events from zbus. */
```

Reply:
I will ignore this as we have about a billion of these comments in our code without periods. I see the point for the Doxygen documentation that will be parsed but not for one liners within the code